Pipfile

```diff
@@ -0,0 +1,16 @@
+[[source]]
```

Comment:
Can we use poetry instead of pipenv please? We've encountered so many problems with pipenv taking an enternity on csvcubed.